spec/components/schema/payment.yaml

```diff
@@ -101,12 +101,13 @@ components:
       example: EUR
     PaymentMethod:
       type: string
-      enum: ["cc", "pp", "dd"]
+      enum: ["cc", "pp", "dd", "mb"]
```

Comment:
Let's not expose this information. It will probably lead to partners who do not qualify for master bill to make many requests to care B2B teams.